digits/scheduler.py

```diff
@@ -166,7 +166,8 @@ def delete_job(self, job):
             job_id = job.id()
         else:
             raise ValueError('called delete_job with a %s' % type(job))
-
+        DependencyExists = False
+        ErrorDetails = ''
```

Comment:
Lower case variable names, please.

https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables